python/test/Ice/operations/ServerAMD.py

```diff
@@ -9,12 +9,7 @@
 if "--load-slice" in sys.argv:
     TestHelper.loadSlice("Test.ice")
 
-# TODO: This is no longer relevant. All supported Python versions provide async/await keywords.
-# We want to test coroutines but older versions of Python cannot
-# load a source file that uses the async/await keywords, so we
-# have two versions of MyDerivedClassI.
-#
-from TestAMDCoroI import MyDerivedClassI
+from TestAMDI import MyDerivedClassI
```

Comment:
Removed the comment, and renamed TestAMDCoroI to TestAMDI, removed the old TestAMDI which was no longer used.